gsplat/cuda/_wrapper.py

```diff
@@ -395,6 +396,7 @@ def rasterize_to_pixels(
         isect_offsets: Intersection offsets outputs from `isect_offset_encode()`. [C, tile_height, tile_width]
         flatten_ids: The global flatten indices in [C * N] or [nnz] from  `isect_tiles()`. [n_isects]
         backgrounds: Background colors. [C, channels]. Default: None.
+        masks: Optional masks to support Grendel's local workload strategy. [C, tile_height, tile_width]. Default: None.
```

Comment:
Same here, would be quite confused to user if the user is not familiar with grendel. Should just say "Optional tile mask to skip render GS to some tiles."

Reply:
fixed